overlay_gl/CMakeLists.txt

```diff
@@ -92,6 +94,9 @@ if(NOT APPLE)
 			# install 32bit overlay library
 			install(TARGETS overlay_gl_x86 LIBRARY DESTINATION "${MUMBLE_INSTALL_LIBDIR}")
 		endif()
+	elseif(${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
+		find_pkg(gl REQUIRED)
+		target_include_directories(overlay_gl PRIVATE ${gl_INCLUDE_DIRS})
```

Comment:
I am wondering why this is only required on OpenBSD - to me it would make a lot of sense if this was also required on e.g. Linux :eyes: 

@davidebeatrici do you know the reason?

Reply:
I had expected an explicit dependency as well there...

Reply:
On Linux the GL folder is in `/usr/include` (which is usually present by default in the compiler's search path list): https://packages.debian.org/bullseye/amd64/libgl-dev/filelist

On OpenBSD the location is `/usr/X11R6/include/GL`, see https://unix.stackexchange.com/questions/441070/why-isnt-there-a-usr-include-gl-directory-on-openbsd.

Reply:
Alright. In that case I'd vote for still executing this part of the cmake code for Linux as well - that seems to make the intent more clear than implicitly relying on files appearing in the correct directory by default.

Reply:
I agree, there should be no issues on other operating systems as long as they provide the relevant `pkg-config` file.